src/Plugin/Visualisation/VisualisationBase.php

```diff
@@ -80,6 +87,7 @@ public function __construct(array $configuration, $plugin_id, $plugin_definition
     $this->stylePluginManager = $style_plugin_manager;
     $this->source = $configuration['source'];
     $this->style = $configuration['style'];
+    $this->entity = $configuration['entity'];
```

Comment:
As I mentioned in one of my previous comments, this should be an optional thing if we want DVF to work with non-entity data sources.